### PR TITLE
Revised Crashed-Probe

### PR DESCRIPTION
**Pull request by ROMVoid95**
a member of the GCC Discord created this new style for the crashed-probe. Would give Venus a nice little upgrade on the planets  features.

Also gave the block directional placement so the spawned probes would have some variation when spawned

### CODE REVIEW

File: src/main/java/micdoodle8/mods/galacticraft/planets/venus/blocks/BlockCrashedProbe.java [hunk 1]
```diff
@@ -112,4 +114,43 @@ private void spawnItem(World worldIn, BlockPos pos)
         entityitem.setDefaultPickupDelay();
         worldIn.spawnEntity(entityitem);
     }
+    
+        @Override
+    public void onBlockPlacedBy(World worldIn, BlockPos pos, IBlockState state, EntityLivingBase placer, ItemStack stack)
```
Comment (marcus8448) [inline src/main/java/micdoodle8/mods/galacticraft/planets/venus/blocks/BlockCrashedProbe.java:119]:
```suggestion
    @Override
    public void onBlockPlacedBy(World worldIn, BlockPos pos, IBlockState state, EntityLivingBase placer, ItemStack stack)
```

**Comment by EzerArch** [comment]
Please, add some screenshots :)

**Comment by ROMVoid95** [comment]
hey @micdoodle8 ;)

**Comment by micdoodle8** [comment]
Wow, looks great. Thank you :)

**Pull request merged**